src/learning/TowerOfDerivatives.ts

```diff
@@ -0,0 +1,23 @@
+export type TowerOfDerivatives = {
+  // The set of parameters at which the objective function was evaluated
+  xs: number[][];
```

Comment:
There's nothing wrong here, just wanted to point something out about TypeScript and how it handles comments.

In types/interfaces/classes, single line comments aren't picked up by the documentation generator, but multiline comments _are_.

If you were to change your comments to be formatted like the following, then these fields on this type would show these comments when hovering over those fields anywhere else in the code (otherwise, nothing shows other than the type of the field).

```ts
export type TowerOfDerivatives = {
  /*
   * The set of parameters at which the objective function was evaluated
   */
  xs: number[][];

  // ...
}
```

Reply:
There's also an eslint rule that you could use (temporarily) to auto-fix these single line comments to be multilines; I can show you how to do that / do that for you if you'd like these comments updated.